mne/decoding/transformer.py

```diff
@@ -138,14 +203,28 @@ def inverse_transform(self, epochs_data, y=None):
                              % type(epochs_data))
 
         X = np.atleast_3d(epochs_data)
+        n_epochs, n_channels, n_times = X.shape
+        X = X.transpose(1, 0, 2)
 
-        for key, this_pick in six.iteritems(self.picks_list_):
-            if self.with_mean:
-                X[:, this_pick, :] += self.ch_mean_[key]
-            if self.with_std:
-                X[:, this_pick, :] *= self.std_[key]
+        if self.scalings == 'standard':
```

Comment:
I would simplify this 'standard' method by just using sklearn.preprocessing.StandardScaler(), and collect mean_ and std from Scaler().mean_ Scaler().std_ by adding a property. The code should be pretty simple then.